data/human/jobs.txt

```diff
@@ -2737,6 +2737,251 @@ mission "Bounty Hunting (Small, Boarding, Entering)"
 		payment 175000
 		dialog "The government of <planet> gratefully pays you <payment> for apprehending the criminal."
 
+mission "Bounty Hunting (Small, Hidden)"
+	name "Disguised bandit near <system>"
+	description "A low-end pirate warship named the <npc> has masqueraded as a merchant ship near the <system> system to avoid detection. Select merchant ships until you find a ship with a matching name, then destroy it and return to <planet> for payment (<payment>)."
```

Comment:
```suggestion
	description "A low-end pirate warship named the <npc> is masquerading as a merchant ship near the <system> system to avoid detection. Select merchant ships until you find a ship with a matching name, then destroy it and return to <planet> for payment (<payment>)."
```
I think either tense is usable here, but (to the best of my knowledge) the norm in-game is present tense for these types of things.